packages/babel-core/src/transformation/file/index.js

```diff
@@ -52,12 +52,12 @@ export default class File extends Store {
     this.opts = this.initOptions(opts);
 
     this.parserOpts = {
-      highlightCode: this.opts.highlightCode,
-      nonStandard:   this.opts.nonStandard,
-      sourceType:    this.opts.sourceType,
-      filename:      this.opts.filename,
-      sourceFileName:this.opts.filename,
-      plugins:       []
+      allowImportExportEverywhere: this.opts.allowImportExportEverywhere,
+      allowReturnOutsideFunction:  this.opts.allowReturnOutsideFunction,
+      allowSuperOutsideMethod:     this.opts.allowSuperOutsideMethod,
```

Comment:
Yep I was going to mention that too - if there are invalid might as well not use them and get everyone to use `parserOpts` instead